ironmon_tracker/screens/TrackerScreen.lua

```diff
@@ -6,7 +6,8 @@ TrackerScreen.Buttons = {
 		getIconPath = function(self)
 			local pokemonID = 0
 			local pokemon = Tracker.getViewedPokemon()
-			if pokemon ~= nil and PokemonData.isValid(pokemon.pokemonID) then
+			--Don't want to consider Ghost a valid pokemon, but do want to use its ID (413) for the image name
+			if pokemon ~= nil and (PokemonData.isImageIDValid(pokemon.pokemonID)) then
```

Comment:
See PokemonData comment, Ghost is not a valid pokemon BUT the ID is a valid image